.github/workflows/nebula.yml

```diff
@@ -26,14 +26,18 @@ jobs:
     name: Gradle Build without Publish
     steps:
       - uses: actions/checkout@v1
-      - name: Set up JDK 1.8
-        uses: actions/setup-java@v1
+      - name: Set up JDK 8
+        uses: actions/setup-java@v2
         with:
-          java-version: 1.8
+          distribution: 'zulu'
```

Comment:
we were building with temuring... so upgraded to v2 of the action to set the vendor